gstreamer-rtsp-server/src/auto/r_t_s_p_server.rs

```diff
@@ -0,0 +1,335 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir @ 9ae0a4a+)
+// from gir-files (https://github.com/gtk-rs/gir-files @ ???)
+// DO NOT EDIT
+
+use RTSPAuth;
+use RTSPClient;
+use RTSPMountPoints;
+use RTSPSessionPool;
+use RTSPThreadPool;
+use ffi;
+use gio;
+use glib;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct RTSPServer(Object<ffi::GstRTSPServer, ffi::GstRTSPServerClass>);
+
+    match fn {
+        get_type => || ffi::gst_rtsp_server_get_type(),
+    }
+}
+
+impl RTSPServer {
+    pub fn new() -> RTSPServer {
+        assert_initialized_main_thread!();
+        unsafe {
+            from_glib_full(ffi::gst_rtsp_server_new())
+        }
+    }
+
+    pub fn io_func(socket: &gio::Socket, condition: glib::IOCondition, server: &RTSPServer) -> bool {
+        skip_assert_initialized!();
+        unsafe {
+            from_glib(ffi::gst_rtsp_server_io_func(socket.to_glib_none().0, condition.to_glib(), server.to_glib_none().0))
+        }
+    }
+}
+
+impl Default for RTSPServer {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+unsafe impl Send for RTSPServer {}
+unsafe impl Sync for RTSPServer {}
+
+pub trait RTSPServerExt {
+    //fn client_filter<'a, P: Into<Option<&'a /*Unimplemented*/RTSPServerClientFilterFunc>>, Q: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, func: P, user_data: Q) -> Vec<RTSPClient>;
+
+    //fn create_socket<'a, P: Into<Option<&'a gio::Cancellable>>>(&self, cancellable: P, error: /*Ignored*/Option<Error>) -> Option<gio::Socket>;
+
+    //fn create_source<'a, P: Into<Option<&'a gio::Cancellable>>>(&self, cancellable: P, error: /*Ignored*/Option<Error>) -> Option<glib::Source>;
+
+    fn get_address(&self) -> Option<String>;
+
+    fn get_auth(&self) -> Option<RTSPAuth>;
+
+    fn get_backlog(&self) -> i32;
+
+    fn get_bound_port(&self) -> i32;
+
+    fn get_mount_points(&self) -> Option<RTSPMountPoints>;
+
+    fn get_service(&self) -> Option<String>;
+
+    fn get_session_pool(&self) -> Option<RTSPSessionPool>;
+
+    fn get_thread_pool(&self) -> Option<RTSPThreadPool>;
+
+    fn set_address(&self, address: &str);
+
+    fn set_auth<'a, P: Into<Option<&'a RTSPAuth>>>(&self, auth: P);
+
+    fn set_backlog(&self, backlog: i32);
+
+    fn set_mount_points<'a, P: Into<Option<&'a RTSPMountPoints>>>(&self, mounts: P);
+
+    fn set_service(&self, service: &str);
```

Comment:
Getter and setter disagree about nullability

Reply:
legit though :(